README.md

```diff
@@ -37,13 +37,16 @@ In `wp-config.php` or a similarly-early and appropriate place, define `CRON_CONT
 ```
 define( 'CRON_CONTROL_ADDITIONAL_INTERNAL_EVENTS', array(
 array(
-	'schedule' => 'hourly',
-	'action'   => 'do_a_thing',
-	'callback' => '__return_true',
+	'schedule'  => 'hourly',
+	'action'    => 'do_a_thing',
+	'callback'  => '__return_true',
+	'timestamp' => strtotime( '+60 seconds' ),
 ),
 ) );
 ```
 
+The `timestamp` parameter is optional, and will default to +60 seconds if left off. Useful in cases where you would like to stagger the intial run of a new internal event across sites.
```

Comment:
Should this read "will default to JOB_QUEUE_WINDOW_IN_SECONDS from now"?

Reply:
Also, "initial" I presume

Reply:
True, but since `JOB_QUEUE_WINDOW_IN_SECONDS` defaults to +60 seconds it's not lying :), and that felt more useful in the example.